### DISCUSSION TITLE
What is the most recent BB version that allows nested forms?

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#11258] [Help] [answered]
Recently I found out that nested forms are prohibited by new versions of BB by a validation logic. However, my use case requires the usage of nested forms, to be able to communicate and share data between child forms.
So I need to revert my self-host BB back to the latest version with nested forms allowed. Could anyone point me which is the version I'm looking for?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @dannyshin76,

You should be able to use **v2.7.36**, I'm pretty sure this was added during the June release which was v2.8.0. If **2.7.36** still has it which I'm pretty sure it doesn't going 1 additional version back will definitely not have it. (2.7.33)

I hope this helps and makes sense.

**Reply by dannyshin76** [reply]
Got that, thanks!
